package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chainsafe/dappeteer",
-  "version": "2.4.1",
+  "version": "3.0.0-next.0",
```

Comment:
```suggestion
  "version": "3.0.0-rc.0",
```

Reply:
I don't think it is ready for release (and for not having breaking changes) but it is in better shape than 2xx that is the rationale behind `beta`, should we still process with `rc`?

Reply:
yeah, it's release candidate, doesn't mean it's ready to be promoted